axiom/nr_txn.h

```diff
@@ -468,6 +468,7 @@ extern nr_status_t nr_txn_record_error_worthy(const nrtxn_t* txn, int priority);
 
 extern void nr_txn_record_error(nrtxn_t* txn,
                                 int priority,
+                                bool add_to_segment,
```

Comment:
Please update doc string with explanation of the new parameter.

Reply:
updated in [bce3e14](https://github.com/newrelic/newrelic-php-agent/pull/767/commits/bce3e1462db26bb5f841542c199dae92ba96a1b3)